### DISCUSSION TITLE
Notify @here or @everyone in Discord

### DISCUSSION THREAD

**Discussion by SolomonHD** [containrrr/shoutrrr#379] [Q&A]
Hello,

Is it possible to notify a Discord channel with <span>@</span>everyone or <span>@</span>here with shoutrrr?

I've tried "<span>@</span>everyone" and "<<span>@</span>everyone>" and it's just printed the words without the channel wide notify.

**Comment by piksel** [comment]
No, it needs some special support in the API. It has been discussed before here: https://github.com/containrrr/shoutrrr/discussions/295
